src/components/CvDatePicker/CvDatePicker.stories.js

```diff
@@ -237,11 +237,32 @@ SingleUsingDate.parameters = storyParametersObject(
 );
 
 /* SINGLE USING MIN MAX PARAMS STORY */
-
+const calOptions = ref({
+  minDate: now,
+  maxDate: nextWeek,
+  dateFormat: 'm/d/Y',
+});
+function toggleDateFormat() {
+  if (calOptions.value.dateFormat === 'm/d/Y')
```

Comment:
Nice idea. I do not actually see any constants in flatpickr but I like the idea of adding constants to the component.
I really like the luxon (formerly moment) library and they define a few really useful presets.
https://moment.github.io/luxon/#/formatting?id=presets

Most of them we could define but the very first in the list is tricky since it toggles between "m/d/Y" and "d/m/Y" based on the locale.